### ISSUE TITLE
table is not configured with charset utf8mb4 -- This may lead to crashes when certain characters are pasted in pads

### ISSUE THREAD

**Issue by vivekvt** [ether/etherpad-lite#4234]
I am using AWS RDS Mysql. 

Everything is working fine I can create pad. **Etherpad is Awesome**
settings.json
```
 "dbType": "mysql",
  "dbSettings": {
    "user": "<user>",
    "host": "<host>",
    "port": "3306",
    "password": "<password>",
    "database": "etherpad_db",
    "charset": "utf8mb4"
  },
```

But whenever I start 
`./bin/run.sh
`

I get this Error

**[2020-08-28 23:05:43.828] [ERROR] console - table is not configured with charset utf8mb4 -- This may lead to crashes when certain characters are pasted in pads**

![Screenshot from 2020-08-28 23-16-07](https://user-images.githubusercontent.com/50275510/91599803-7c626700-e984-11ea-94b1-d1463927a3c7.png)

 - OS: Ubuntu 18.04.5 LTS
 - Browser Google Chrome
 - Version Version 84.0.4147.125

**Comment by JohnMcLear** [comment]
Good.  Etherpad is doing its job.  Read the Amazon docs.  Amazon MySQL is not supported by ueberdb

**Comment by brunob** [comment]
This one could be closed, no ?

**Comment by JohnMcLear** [comment]
@vivekvt Amazon is the problem here, not Etherpad.  Amazon MySQL (whatever the fuck that is) needs the correct charset.  Learn how to set that in the Amazon whatever.  We don't support Amazon or Azure or any other corporate (closed) offering.  We only support "MySQL" in a pure, unadulterated form.  If you want to use Amazon MySQL (assuming it's MySQL or Maria) then that's up to you to support it.

TLDR; we support open standards, any bastardization is not our problem.
